spec/space_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'space'
+
+describe Space do
+  it "shows available spaces" do
+    spaces = Space.all
+
+    expect(spaces.length).to eq 3
+    expect(spaces.first).to be_a Space
+    expect(spaces.first.id).to eq "1" #dodgy because id will increment later- currently do not have a .create method to rectify this
```

Comment:
As you commented, we can change this once we merge to use the create method from the other branch to avoid the janky branch

`space = Space.create(name: "Space1", description: "Description1", price: 1)
expect(spaces.first.id) = space.id

`